src/Dotnet.Script.Desktop.Tests/CompilationDepenencyTests.cs

```diff
@@ -14,13 +14,15 @@ public CompilationDependencyTests(ITestOutputHelper testOutputHelper)
             testOutputHelper.Capture();
         }
 
-        [Fact]
-        public void ShouldGetCompilationDependenciesForNetCoreApp2_1()
+        [Theory]
+        [InlineData("netcoreapp2.1")]
+        [InlineData("netcoreapp3.0")]
+        public void ShouldGetCompilationDependenciesForNetCoreApp2_1(string targetFramework)
```

Comment:
we should rename this test as it now tests both targets

Reply:
👍